schemas/common/url.cue

```diff
@@ -13,4 +13,4 @@
 
 package common
 
-#url: =~"^https?:\\/\\/[a-z]+(?:\\.[a-z]+)*(:\\d+)?(?:\\/[a-z\\d]+)*$"
+#url: =~"^https?:\/\/[^\\s\/$.?#].[^\\s]*$"
```

Comment:
it looks like the previous regex was escaping twice the / with \\/ but now it's \/ is that change wanted?